components/mpas-ocean/src/mode_init/mpas_ocn_init_global_ocean.F

```diff
@@ -525,6 +503,33 @@ subroutine ocn_init_setup_global_ocean(domain, iErr)!{{{
          call mpas_dmpar_finalize(domain % dminfo)
       end if
 
+      call mpas_log_write( 'Compute density')
+       block_ptr => domain % blocklist
+       do while(associated(block_ptr))
+
+          call mpas_pool_get_subpool(block_ptr % structs, 'mesh', meshPool)
+          call mpas_pool_get_subpool(block_ptr % structs, 'state', statePool)
+          call mpas_pool_get_dimension(meshPool, 'nCells', nCells)
+          call mpas_pool_get_array(meshPool, 'minLevelCell', minLevelCell)
+
+          do iCell = 1, nCells
+             do iTracer = 1,size(tracersSurfaceValue,1)
+                tracersSurfaceValue(iTracer, iCell) = activeTracers(iTracer, minLevelCell(iCell), iCell)
+             enddo
+          end do
+
+          call ocn_equation_of_state_density(statePool, meshPool, tracersSurfaceValue, &
+             nCells, 0, 'relative', density, iErr, &
+             timeLevelIn=1)
```

Comment:
Am I right in thinking that this density calculation doesn't use land ice pressure (atm pressure everywhere)? So as far as this calculation is concerned, ice shelf cavities don't exist yet (ssh = 0)?

Reply:
It uses a reference pressure profile that doesn't vary at all with space.  It doesn't even use atmospheric pressure.  (It's really kind of an embarrassment).